### PR TITLE
maintainers-guide: refresh

### PR DESCRIPTION
**Pull request by CleanMachine1**
There are a few issues here, mostly things that are outdated from our discussions of the review process.

### CODE REVIEW

File: contributing-guides/maintainers-guide.md [hunk 1]
```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
   (2) have the **review comments addressed**, and
   (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (4) are at least **24 hours** old unless changes are trivial
```
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:61]: Feel free to change the value, I've been using 24 hours but that is just me.

File: contributing-guides/maintainers-guide.md [hunk 2]
```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
...
+  the first maintainer should ping others for review.
 
 - If the only issues holding up a merge are **trivial fixes**
   (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
   **maintainers can make the necessary changes themselves**,
   and proceed with the merge process.
 
 - If a PR **stops getting feedback from the submitter** and is marked as stale
-  by [probot-stale](../.github/stale.yml),
+  by [probot-stale](../.github/workflows/stale.yml),
```
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:72]: This error was just because it was outdated.

File: contributing-guides/maintainers-guide.md (hunk 1 above)
Comment (marchersimon) [inline contributing-guides/maintainers-guide.md:61]: I think 24 hours is a good rule of thumb. I'd rephrase this a little though.
```suggestion
  (2) have the **review comments addressed**,
  (3) get **approved reviews by two maintainers**, and
  (4) have been open for at least **24 hours** unless the changes are trivial
```

Comment (sbrl) [reply contributing-guides/maintainers-guide.md:61]: I would suggest that both of these items are good to have here.

File: contributing-guides/maintainers-guide.md [hunk 3]
```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
...
   (2) have the **review comments addressed**, and
   (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (4) are at least **24 hours** old unless changes are trivial
 
 - If a PR fails to get a review from a second maintainer after a few days,
-  the first maintainer should ping others for review. If it still lingers around
-  for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
+  the first maintainer should ping others for review.
```
Comment (sbrl) [inline contributing-guides/maintainers-guide.md:64]: This should stay as-is.

Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:64]: PRs need 2 approvals so it is wrong.

**Review by blueskyson — APPROVED** [review]

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (navarroaxel) [reply contributing-guides/maintainers-guide.md:64]: @CleanMachine1 This clause gives the power to org owners to force merge skipping that PR check.

File: contributing-guides/maintainers-guide.md (hunk 1 above)
Comment (navarroaxel) [reply contributing-guides/maintainers-guide.md:61]: Looks like Sbrl keep the explanation that
> the second maintainer can perform the merge immediately after accepting.
Leaving explicit that after the 2nd approve the maintainer shouldn't wait another stuff.

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:64]: But should we (you) even do that? We've never had a PR which had to be merged that urgently but didn't have 2 approvals.

File: contributing-guides/maintainers-guide.md (hunk 1 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:61]: @sbrl reffering to your comment [here](https://github.com/tldr-pages/tldr/pull/6397#discussion_r693549872), what do you think about this:
```suggestion
  (2) have the **review comments addressed**,
  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
  (4) have been open for at least **24 hours** unless the changes are trivial
```

Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:61]: @CleanMachine1 could you please apply this? ⬆️

Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:61]: @marchersimon Could you please resend the suggestion or feel free to edit the page, I can't see whats being changed.

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:64]: So what should we do about this? Leave it?

File: contributing-guides/maintainers-guide.md (hunk 1 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md]: Done in ba415df

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:64]: Maybe it can be phrased to show that only in occasions such as bug testing and trivial changes would be acceptable

Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:64]: I think even trivial changes should be approved by at least two maintainers. The rule isn't really relevant, so I think we can just leave it.

Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:64]: @sbrl could we get your reply about this subject.

Comment (sbrl) [reply contributing-guides/maintainers-guide.md:64]: > But should we (you) even do that? We've never had a PR which had to be merged that urgently but didn't have 2 approvals.

Urgently? No, not really - usually posting urgent urgent urgent ones in Gitter is enough. There have been plenty of times when I've force-merged for example an obvious typo though. Generally speaking I only force-merge uncontroversial changes in periods of very high activity to avoid too much notification spam.

Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:64]: Hm, I never noticed that, but it makes sense. I think we can keep the original, but I don't have a strong opinion here.

File: contributing-guides/maintainers-guide.md [hunk 4]
```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**, and
-  (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (2) have the **review comments addressed**,
+  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
```
Comment (xeruf) [inline contributing-guides/maintainers-guide.md:60]:
```suggestion
  (3) are **approved by two maintainers**
      (the second maintainer can merge immediately after approving)
```

File: contributing-guides/maintainers-guide.md [hunk 5]
```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
```
Comment (xeruf) [inline contributing-guides/maintainers-guide.md:57]:
```suggestion
- PRs should be merged once they pass all following points:
```

File: contributing-guides/maintainers-guide.md [hunk 6]
```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
...
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**, and
-  (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (2) have the **review comments addressed**,
```
Comment (xeruf) [inline contributing-guides/maintainers-guide.md:59]:
```suggestion
  (2) all **review comments are addressed**
```

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (xeruf) [reply contributing-guides/maintainers-guide.md:64]: I would side with the modification, this practice does not concern typical maintainers and we should strive to cut down noise.

**Review by xeruf — CHANGES_REQUESTED** [review]

File: contributing-guides/maintainers-guide.md (hunk 5 above)
Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:57]: I disagree.

Comment (xeruf) [reply contributing-guides/maintainers-guide.md:57]: do you disagree with the content or the wording? It makes a previously implicit point explicit, reducing reader friction

Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:57]: I disagree with the content. I think it interrupts the reader's flow a bit.

**Comment by sbrl** [comment]
> I would side with the modification, this practice does not concern typical maintainers and we should strive to cut down noise.

Could you define "typical maintainer" for me please? Note that the maintainer's guide should be the definitive guide for *all* maintainers - regardless of experience level.

File: contributing-guides/maintainers-guide.md (hunk 3 above)
Comment (sbrl) [reply contributing-guides/maintainers-guide.md:64]: I'm not sure what you mean by "typical maintainer" here @xeruf. Note that the maintainer's guide should be the definitive guide for *all* maintainers - regardless of experience level.

File: contributing-guides/maintainers-guide.md (hunk 4 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:60]: I agree with `get approved reviews` → `are approved`, but I don't understand the newline.

File: contributing-guides/maintainers-guide.md (hunk 6 above)
Comment (marchersimon) [reply contributing-guides/maintainers-guide.md:59]: This would only make sense with the change above, so I'll resolve this to keep the conversation cleaner.

File: contributing-guides/maintainers-guide.md [hunk 7]
```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
...
+  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
+  (4) have been open for at least **24 hours** unless the changes are trivial
 
 - If a PR fails to get a review from a second maintainer after a few days,
-  the first maintainer should ping others for review. If it still lingers around
-  for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
+  the first maintainer should ping others for review.
```
Comment (marchersimon) [inline contributing-guides/maintainers-guide.md:64]: As discussed above, I think we should revert this change.
```suggestion
  the first maintainer should ping others for review. If it still lingers around
  for **over a week without a second maintainer’s approval**,
  the first maintainer can go ahead and merge it.
```

File: contributing-guides/maintainers-guide.md (hunk 4 above)
Comment (xeruf) [reply contributing-guides/maintainers-guide.md:60]: better readability. On my screen at least I cannot see the whole line in your proposal.
https://sembr.org
